package.json

```
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "cypress:test": "npm run back-end:build && npm run back-end:start & wait-on --timeout 60000 http://localhost:3000 & ./node_modules/.bin/cypress run",
+    "build": "npm run back-end:build & npm run front-end:build",
```

We don't need to daemonize the backend build, right?
```suggestion
    "build": "npm run back-end:build && npm run front-end:build",
```

---

Nope, just a typo